src/main/java/com/sap/sgs/phosphor/fosstars/data/github/NvdEntryMatcher.java

```diff
@@ -95,31 +114,27 @@ public boolean match(NvdEntry entry) {
       return false;
     }
 
-    return match(cve.getAffects());
+    return match(cve.getAffects(), project);
   }
 
   /**
-   * Returns true if one of the configurations matches the project, false otherwise.
+   * Returns true if one of the configurations of type {@link Configurations} matches the project.
+   * Also checks if the references of type {@link ReferenceLink} from {@link CVE} contains any
+   * useful information. Otherwise false.
```

Comment:
Let's please make it simpler.

```suggestion
   * Checks if one of the {@link Configurations} matches the project.
   * Also checks if any {@link ReferenceLink} from {@link CVE} refers to the project.
```

We're also trying to have complete javadoc. It would be good if you added `param`, `returns`, etc tags.